### PR TITLE
Charts raw wrapper

### PR DESCRIPTION
**Pull request by Starzu**

### CODE REVIEW

**Comment by Starzu** [comment]
Udash Guide examples: https://github.com/UdashFramework/udash-guide/commit/e38bc0284152eef042904dc54e0e1b36d801b736

File: charts/frontend/src/main/scala/io/udash/wrappers/highcharts/config/tooltip/Tooltip.scala [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+  val xDateFormat: js.UndefOr[String] = js.undefined
+}
+
+object Tooltip {
+  import scala.scalajs.js.JSConverters._
+
+  @js.native
+  trait FormatterData extends js.Object {
+    val percentage: Double = js.native
+    val point: Point = js.native
```
Comment (ddworak) [inline charts/frontend/src/main/scala/io/udash/wrappers/highcharts/config/tooltip/Tooltip.scala]: I'm afraid one of {`point`, `points`} is undefined depending on the plurality, just Highcharts things...

**Comment by ddworak** [comment]
I think I've seen enough of it in action to accept, could you only verify the unverified (`TODO not verified`)?

**Comment by Starzu** [comment]
@ddworak Done and squashed.

**Pull request merged**